dlp/snippets/deid/deid_table_fpe.go

```diff
@@ -0,0 +1,171 @@
+			{Name: "Employee ID"},
+			{Name: "Date"},
+			{Name: "Compensation"},
+		},
+		Rows: []*dlppb.Table_Row{
+			{Values: row1.Values},
+			{Values: row2.Values},
+			{Values: row3.Values},
+		},
+	}
+
+	// client operations.
+	ctx := context.Background()
+
+	// Initialize a client once and reuse it to send multiple requests. Clients
+	// are safe to use across goroutines. When the client is no longer needed,
+	// call the Close method to cleanup its resources.
+	client, err := dlp.NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("dlp.NewClient: %v", err)
+	}
+
+	// Closing the client safely cleans up background resources.
+	defer client.Close()
+
+	// Specify what content you want the service to de-identify.
+	contentItem := &dlppb.ContentItem{
+		DataItem: &dlppb.ContentItem_Table{
+			Table: table,
+		},
+	}
+
+	// Specify an encrypted AES-256 key and the name of the Cloud KMS key that encrypted it.
+	kmsKeyDecode, err := base64.StdEncoding.DecodeString(wrappedAesKey)
+	if err != nil {
+		return fmt.Errorf("error in decoding key: %v", err)
+	}
+
+	kmsWrappedCryptoKey := &dlppb.KmsWrappedCryptoKey{
+		WrappedKey:    kmsKeyDecode,
+		CryptoKeyName: kmsKeyName,
+	}
+
+	cryptoKey := &dlppb.CryptoKey_KmsWrapped{
+		KmsWrapped: kmsWrappedCryptoKey,
+	}
+
+	// Specify how the content should be encrypted.
+	cryptoReplaceFfxFpeConfig := &dlppb.CryptoReplaceFfxFpeConfig{
+		CryptoKey: &dlppb.CryptoKey{
+			Source: cryptoKey,
+		},
+		Alphabet: &dlppb.CryptoReplaceFfxFpeConfig_CommonAlphabet{
+			CommonAlphabet: dlppb.CryptoReplaceFfxFpeConfig_NUMERIC,
+		},
+	}
+
+	primitiveTransformation := &dlppb.PrimitiveTransformation{
+		Transformation: &dlppb.PrimitiveTransformation_CryptoReplaceFfxFpeConfig{
+			CryptoReplaceFfxFpeConfig: cryptoReplaceFfxFpeConfig,
+		},
+	}
+
+	// Specify field to be encrypted.
+	fieldId := &dlppb.FieldId{
+		Name: "Employee ID",
+	}
+
+	// Associate the encryption with the specified field.
+	fieldTransformation := &dlppb.FieldTransformation{
+		Fields: []*dlppb.FieldId{
+			fieldId,
+		},
+		Transformation: &dlppb.FieldTransformation_PrimitiveTransformation{
+			PrimitiveTransformation: primitiveTransformation,
+		},
+	}
+
+	transformations := &dlppb.RecordTransformations{
+		FieldTransformations: []*dlppb.FieldTransformation{
+			fieldTransformation,
+		},
+	}
+
+	// Construct the de-identification request to be sent by the client.
+	req := &dlppb.DeidentifyContentRequest{
+		Parent: fmt.Sprintf("projects/%s/locations/global", projectID),
+		DeidentifyConfig: &dlppb.DeidentifyConfig{
+			Transformation: &dlppb.DeidentifyConfig_RecordTransformations{
+				RecordTransformations: transformations,
+			},
+		},
+		Item: contentItem,
+	}
+
+	// Send the request.
+	resp, err := client.DeidentifyContent(ctx, req)
+	if err != nil {
+		return fmt.Errorf("DeidentifyContent: %v", err)
+	}
+
+	// Print the results.
+	fmt.Fprintf(w, "De-identify Table after format-preserving encryption : %v", resp.GetItem().GetTable())
```

Comment:
Will this come out in a user-intelligible form?

Reply:
- I kind of sound repetitive @tritone, but According to the other samples in Golang and other languages I have put it in a way, but again open to change but just want you to know my thoughts behind it, let me know what should be the best for users I will change according to it. Again Thank you for your suggestions and efforts.

Reply:
@AarshD-crest is correct here. See Samples Style Guide:

https://googlecloudplatform.github.io/samples-style-guide/#result

Reply:
Thank you for clearing that out @tritone, and considering your suggestion In spite of this is a nit, I have compared the results between `%+v` and `%v` but I didn't find any difference between the outputs, They are identical so keeping it as it is. Once again Thank you so much @tritone for your guidance and suggestions.